ext/websocket/stream.rs

```diff
@@ -109,6 +168,10 @@ impl AsyncWrite for WebSocketStream {
       WsStreamKind::Upgraded(stream) => {
         Pin::new(stream).poll_write_vectored(cx, bufs)
       }
+      WsStreamKind::H2(..) => {
+        // TODO(mmastrac): this is possibly just too difficult, but we'll never call it
+        unimplemented!()
+      }
```

Comment:
Are we sure this won't be hit?

Reply:
Yep -- I explicitly disabled the writev flag on fastwebsockets for H2.